hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -285,7 +286,6 @@ public synchronized void start() {
     if (!isRunning()) {
       LOG.info("Starting Replication Monitor Thread.");
       running = true;
-      metrics = ReplicationManagerMetrics.create(this);
       if (rmConf.isLegacyEnabled()) {
         legacyReplicationManager.setMetrics(metrics);
       }
```

Comment:
Thanks @aswinshakil for the fix.  Please note that `ReplicationManager` can be stopped/restarted by admin from CLI.  
`stop()` unregisters the metrics source, and `start()` used to (re)create it.   But now it isn't re-created, so it won't work after restart.  One way to fix this is to add a method in `ReplicationManagerMetrics` to register it with the metrics system (without creating a new instance), and call this new method from `start()`.

Reply:
Thanks for pointing out @adoroszlai . I wasn't aware of that, feel free to assign the JIRA to me if created, or I can create one.

Reply:
Thanks @aswinshakil, created https://issues.apache.org/jira/browse/HDDS-9235.